dataland-e2etests/src/test/kotlin/org/dataland/e2etests/MetaDataControllerTest.kt

```diff
@@ -103,10 +89,20 @@ class MetaDataControllerTest {
     fun `post companies and eu taxonomy data and check meta info search with filters on company ID and data type`() {
         val numberOfCompanies = 2
         val numberOfDataSetsToPostPerCompany = 6
-        val testData = generateTestData(numberOfCompanies, numberOfDataSetsToPostPerCompany, 4000, 500000000)
+        val testData = testDataProvider.getCompaniesWithData(numberOfCompanies, numberOfDataSetsToPostPerCompany)
         val listOfTestCompanyIds = postCompaniesAndEuTaxonomyData(testData)
         val listOfDataMetaInfoPerCompanyIdAndDataType =
             metaDataControllerApi.getListOfDataMetaInfo(listOfTestCompanyIds.first(), "EuTaxonomyData")
         assertEquals(numberOfDataSetsToPostPerCompany, listOfDataMetaInfoPerCompanyIdAndDataType.size)
     }
+
+    @Test
+    fun `post companies and eu taxonomy data and check if green asset ratio is in expected range`() {
+        val numberOfCompanies = 2
+        val numberOfDataSetsToPostPerCompany = 6
+        val testData = testDataProvider.getCompaniesWithData(numberOfCompanies, numberOfDataSetsToPostPerCompany)
+        postCompaniesAndEuTaxonomyData(testData)
+        val greenAssetRatio = metaDataControllerApi.getGreenAssetRatio(null)
+        assertTrue(greenAssetRatio.all { it.value.toDouble() in 0.0..1.0 })
```

Comment:
Error message hinzufügen. Z.B. "At least one Green Asset Ratio is out of bounds."

Reply:
Ich finde den Test eigentlich recht sprechend und die anderen Tests in dem Skript sind auch ohne Message. Es gibt jetzt auch keine Information die man sinnvollerweise zusätzlich in dei message packen könnte außer halt das mindestens einer out of range ist.

Reply:
Ok in den anderen E2E-Test-Klassen haben wir aber ausnahmslos error messages geschrieben.
Die haben mir teilweise auch schon geholfen.
Beim green asset ratio stimme ich dir zu, dass man nicht wirklich hiflreiche Informationen in die message packen kann.
Aber bei den anderen Tests hier in der MetaDataControllerTest-Klasse würde es z.B. helfen, die Anzahl der Suchergebnisse, die bei einem Fail nicht wie erwartet sind, in die Error Messages zu packen, oder?

Oder sehe ich das auch ohne error message automatisch im Log?